rootfs/api/models/app.py

```diff
@@ -792,7 +792,10 @@ def _build_env_vars(self, release):
         # mix in default environment information deis may require
         default_env = {
             'DEIS_APP': self.id,
-            'WORKFLOW_RELEASE': 'v{}'.format(release.version)
+            'WORKFLOW_RELEASE': 'v{}'.format(release.version),
+            'WORKFLOW_RELEASE_SUMMARY': release.summary,
+            'WORKFLOW_RELEASE_CREATED_AT': str(release.created.strftime(
+                settings.DEIS_DATETIME_FORMAT))
```

Comment:
I think it does because we do it elsewhere in the codebase. DRF just handles it in the serializer, I think.